Frontend/Pages/KeyStage4Performance.cshtml.cs

```diff
@@ -81,7 +81,14 @@ private void BuildPageModel(GetInformationForProjectResponse projectInformation,
         private static List<KeyStage4> GetLatestThreeResults(List<KeyStage4> keyStage4Performance)
         {
             return keyStage4Performance.Take(3).OrderByDescending(a => a.Year)
-                .Concat(Enumerable.Range(0, 3).Select(_ => new KeyStage4())).Take(3).ToList();
+                .Concat(Enumerable.Range(0, 3).Select(_ => new KeyStage4())).Take(3).Select(c => 
```

Comment:
Hmmm - i wonder if this is the best place to do it, or whether doing it at the mapping level is 🤔 i guess here makes sense, maybe we make it a helper method though and we can make the change across all 3 key stages?